packages/webview_flutter/ios/Classes/FlutterWebView.m

```diff
@@ -228,18 +234,27 @@ - (void)clearCache:(FlutterResult)result {
   }
 }
 
-- (void)applySettings:(NSDictionary<NSString*, id>*)settings {
+// Returns nil when succesfull, or an error message when one or more keys are unknown.
+- (NSString*)applySettings:(NSDictionary<NSString*, id>*)settings {
+  NSMutableArray<NSString*>* unknownKeys = [[NSMutableArray alloc] init];
   for (NSString* key in settings) {
     if ([key isEqualToString:@"jsMode"]) {
       NSNumber* mode = settings[key];
       [self updateJsMode:mode];
     } else if ([key isEqualToString:@"hasNavigationDelegate"]) {
       NSNumber* hasDartNavigationDelegate = settings[key];
       _navigationDelegate.hasDartNavigationDelegate = [hasDartNavigationDelegate boolValue];
+    } else if ([key isEqualToString:@"debuggingEnabled"]) {
```

Comment:
nits: Prefer to escape early e.g.
```objc
if ([key isEqualToString:@"debuggingEnabled"]) {
  continue;
}
```

Reply:
Feels a little inconsistent to only continue here, so I guess if we do this we should add `continue` in all if cases above, in which case all the `else`s are redundant. So we'll end up with:

```objc
if ([key isEqualToString:@"jsMode"]) {
      NSNumber* mode = settings[key];
      [self updateJsMode:mode];
      continue;
    }
    if ([key isEqualToString:@"hasNavigationDelegate"]) {
      NSNumber* hasDartNavigationDelegate = settings[key];
      _navigationDelegate.hasDartNavigationDelegate = [hasDartNavigationDelegate boolValue];
      continue;
    }
    if ([key isEqualToString:@"debuggingEnabled"]) {
      // no-op debugging is always enabled on iOS.
      continue;
    }
    [unknownKeys addObject:key];
```

Alternatively we can keep it as it currently is. Let me know if you have a preference here, one thing that comes to mind is that relying on continue is more error-prone when copy pasting to add a new key(my feeling is that there's higher chance to miss a `continue` than to miss out an `else`).

Reply:
Sure. Let's keep it as it is!